nixos/modules/services/mail/exim.nix

```diff
@@ -94,6 +94,7 @@ in
     systemd.services.exim = {
       description = "Exim Mail Daemon";
       wantedBy = [ "multi-user.target" ];
+      restartTriggers = [ environment.etc."exim.conf".source ];
```

Comment:
Eval fails (see error below).  You need `config.environment.etc."exim.conf".source`  here.

Reply:
Sorry, fixed and rebased